pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -216,6 +216,42 @@
 					<input cvar="r_physicalMapping" type="checkbox" />
 					<p><translate>Material appearance based on physical approximations.</translate></p>
 				</row>
+				
+				<h2><translate>Reflections</translate></h2>
+				<row>
+					<h3><translate>Static reflections</translate></h3>
+					<input cvar="r_reflectionMapping" type="checkbox" />
+					<p><translate>Adds static cubemap reflections on supported surfaces. Requires normal, specular and deluxe mapping. Currently unsupported with physical mapping.</translate></p>
```

Comment:
Suggestion:

```
<p><translate>Adds static cubemap reflections on supported surfaces. Requires normal, specular and deluxe mapping. Currently unsupported with physical mapping.</translate>&nbsp;<translate>Experimental.</translate></p>
```

This is to ad an (commonly translated) “Experimental” mention, like we do with SSAO.

Reply:
Done now.